README.md

```diff
@@ -35,7 +35,15 @@ module.exports =
 , cache: './cache'
 , salt: 'salt'
 , key: 'key'
+, backend: 'mongo'
 , databaseUri: 'mongodb://localhost:27017/darkroom'
+
+// Or use S3
+, backend: 'S3'
+, accessKeyId: 'FOOBARCAFEBABE'
+, secretAccessKey: 'opefkofwpokfeaioajfew/efwkoewkfopw'
+, region: 'eu-west-2'
+, bucket: 'darkroom'
```

Comment:
are these real?

Reply:
Absolutely not

Reply:
Can you imagine how silly you need to be to push legit aws secrets to a public GitHub repo?

Reply:
Answer: this silly https://github.com/clocklimited/Darkroom-api/compare/6fda036e903a077a2ffba2ddf581e13d3f125f51..136e1a756c379198330d02a9d03144bf11ba2a47